workshops/test/test_event.py

```diff
@@ -187,14 +187,14 @@ def test_website_url(self):
             'https://github.com/user-name/repo-name',
             'https://github.com/user-name/repo-name/',
         ]
-        WEBSITE = 'https://user-name.github.io/repo-name'
+        WEBSITE = 'https://user-name.github.io/repo-name/'
```

Comment:
1. We have tests for lack of trailing slash (or its presence), see https://github.com/swcarpentry/amy/pull/510/files#r40770937
2. In this case it's only a representation that we store in the `WEBSITE` variable.
3. The web interface accepts URLs with both trailing slashes and without, but tries to represent them with the trailing slash present
4. The rationale for adding the trailing slash is that GitHub redirects from `https?://user.github.io/repo` to `https?://user.github.io/repo/`, but it does not do that with repositories in `https://github.com/user/repo` format.